apps/console/src/features/roles/pages/role.tsx

```diff
@@ -70,7 +76,7 @@ const RolesPage: FunctionComponent<RolesPagePropsInterface> = (
     } = useRolesList(
         listItemLimit,
         listOffset,
-        filterBy,
+        `audience.value ne ${consoleId}${ filterBy ? ` and ${ filterBy }` : "" }`,
```

Comment:
shall we move this to a const file ?

```
const useRolesListFilterBy = (filterBy: string) => {
    return `audience.value ne ${consoleId}${ filterBy ? ` and ${ filterBy }` : "" }`
}
```

it would be readable and manageable

Reply:
Fixed